test/feature/features/move.feature

```diff
@@ -0,0 +1,155 @@
+Feature: Move test
+
+  Scenario: MOVE KEY RANGE works
+    Given cluster is up and running
+    When I execute SQL on host "coordinator"
+    """
+    ADD SHARDING RULE r1 COLUMNS w_id;
+    ADD KEY RANGE krid1 FROM 1 TO 10 ROUTE TO sh1;
+    """
+    Then command return code should be "0"
+    When I run SQL on host "shard1"
+    """
+    CREATE TABLE xMove(w_id INT, s TEXT);
+    insert into xMove(w_id, s) values(1, '001');
+    """
+    Then command return code should be "0"
+    When I run SQL on host "shard2"
+    """
+    CREATE TABLE xMove(w_id INT, s TEXT);
+    """
+    Then command return code should be "0"
+    When I execute SQL on host "coordinator"
+    """
+    MOVE KEY RANGE krid1 to sh2
+    """
+    Then command return code should be "0"
+    When I run SQL on host "shard2"
+    """
+    SELECT * FROM xMove
+    """
+    Then command return code should be "0"
+    And SQL result should match regexp
+    """
+    001
+    """
+    When I run SQL on host "shard1"
+    """
+    SELECT * FROM xMove
+    """
+    Then command return code should be "0"
+    And SQL result should not match regexp
+    """
+    001
+    """
+
+    Scenario: MOVE KEY RANGE works with many rows
+    Given cluster is up and running
+    When I execute SQL on host "coordinator"
+    """
+    ADD SHARDING RULE r1 COLUMNS w_id;
+    ADD KEY RANGE krid1 FROM 1 TO 10 ROUTE TO sh1;
+    """
+    Then command return code should be "0"
+    When I run SQL on host "shard1"
+    """
+    CREATE TABLE xMove(w_id INT, s TEXT);
+    insert into xMove(w_id, s) values(1, '001');
+    insert into xMove(w_id, s) values(2, '002');
+    insert into xMove(w_id, s) values(3, '003');
+    insert into xMove(w_id, s) values(4, '004');
+    insert into xMove(w_id, s) values(5, '005');
+    """
+    Then command return code should be "0"
+    When I run SQL on host "shard2"
+    """
+    CREATE TABLE xMove(w_id INT, s TEXT);
+    """
+    Then command return code should be "0"
+    When I execute SQL on host "coordinator"
+    """
+    MOVE KEY RANGE krid1 to sh2
+    """
+    Then command return code should be "0"
+    When I run SQL on host "shard2"
+    """
+    SELECT * FROM xMove
+    """
+    Then command return code should be "0"
+    And SQL result should match regexp
+    """
+    .*001(.|\n)*002(.|\n)*003(.|\n)*004(.|\n)*005
+    """
+    When I run SQL on host "shard1"
+    """
+    SELECT * FROM xMove
+    """
+    Then command return code should be "0"
+    And SQL result should not match regexp
+    """
+    .*001(.|\n)*002(.|\n)*003(.|\n)*004(.|\n)*005
+    """
+
+    Scenario: MOVE KEY RANGE works with many tables
+    Given cluster is up and running
+    When I execute SQL on host "coordinator"
+    """
+    ADD SHARDING RULE r1 COLUMNS w_id;
+    ADD KEY RANGE krid1 FROM 1 TO 10 ROUTE TO sh1;
```

Comment:
Can we create another key range `krid2` and move it insead of `krid1`? Also, it will be better to check whole rows on both shards, as well as check that some rows remains on shard after move. 


so, 
```
*check select * on shard1*
*check select * on shard2*

move key range krid2 .... 


*check select * on shard1*
*check select * on shard2*


```